pinot-controller/src/main/java/org/apache/pinot/controller/helix/core/minion/PinotTaskManager.java

```diff
@@ -483,92 +489,34 @@ public void registerTaskGenerator(PinotTaskGenerator taskGenerator) {
     _taskGeneratorRegistry.registerTaskGenerator(taskGenerator);
   }
 
-  /**
-   * Schedules tasks (all task types) for all tables.
-   * It might be called from the non-leader controller.
-   * Returns a map from the task type to the {@link TaskSchedulingInfo} of tasks scheduled.
-   */
-  public synchronized Map<String, TaskSchedulingInfo> scheduleAllTasksForAllTables(@Nullable String minionInstanceTag) {
```

Comment:
I would suggest rather than removing them add a @Deprecated annotation on top, along with the method that they should be using. We can remove this in next release

Reply:
ok, will revert removed methods